controllers/provisioners/eks/helpers.go

```diff
@@ -214,6 +214,21 @@ func (ctx *EksInstanceGroupContext) GetAddedTags(asgName string) []*autoscaling.
 	tags = append(tags, ctx.AwsWorker.NewTag(provisioners.TagInstanceGroupNamespace, instanceGroup.GetNamespace(), asgName))
 	tags = append(tags, ctx.AwsWorker.NewTag(provisioners.TagInstanceGroupName, instanceGroup.GetName(), asgName))
 
+	if instanceGroup.ObjectMeta.Annotations[ClusterAutoscalerEnabledAnnotation] == "true" {
+		tags = append(tags, ctx.AwsWorker.NewTag("k8s.io/cluster-autoscaler/" + clusterName, "owned", asgName))
+		tags = append(tags, ctx.AwsWorker.NewTag("k8s.io/cluster-autoscaler/enabled", "true", asgName))
+
+		for label, labelValue := range instanceGroup.Spec.EKSSpec.EKSConfiguration.Labels {
+			tags = append(tags, ctx.AwsWorker.NewTag("k8s.io/cluster-autoscaler/node-template/label/" + label, labelValue, asgName))
+		}
+
+		for _, taint := range instanceGroup.Spec.EKSSpec.EKSConfiguration.Taints {
+			tagValue := taint.Value + ":" + string(taint.Effect)
+			tag := ctx.AwsWorker.NewTag("k8s.io/cluster-autoscaler/node-template/taint/" + taint.Key, tagValue, asgName)
+			tags = append(tags, tag)
+		}
+	}
+
```

Comment:
PR looks great! just some (very) minor change suggestions just to follow existing code style:

- Can we add `annotations   = instanceGroup.GetAnnotations()` at the top level var declaration and use `annotations[ClusterAutoscalerEnabledAnnotation]` instead of `instanceGroup.ObjectMeta.Annotations`

- Same for `instanceGroup.Spec.EKSSpec.EKSConfiguration.Labels`, you can use `configuration.GetLabels()` instead

- Same for `instanceGroup.Spec.EKSSpec.EKSConfiguration.Taints`, you can use `configuration.GetTaints()`

- Consider using `fmt.Sprintf` instead of string concatenation - it might be a bit slower, but perhaps makes code bit more readable and also removes the need to do `string(taint.Effect)`? e.g. 
```go
tagValue := taint.Value + ":" + string(taint.Effect)
``` 
vs 
```go
tagValue := fmt.Sprintf("%v:%v", taint.Value, taint.Effect)
```